### PR TITLE
requirePaddingNewLinesAfterBlocks: consider IIFE case

### LINKED ISSUE #1555: Issue with 'requirePaddingNewLinesAfterBlocks' inside of IIFEs
I'm running into an issue with `requirePaddingNewLinesAfterBlocks` where there is an IIFE and I'm passing variables into the closure.

Example:
This fails even with `inCallExpressions` set.
```js
(function (a, b, c) {
  ...
})(
  'a',
  'b',
  'c'
)
```

However this is fine: 
```js
(function (a, b, c) {
  ...
})('a', 'b', 'c')
```
The problem with just doing it that way is I'm running into line length issues when it's all on a single line. 

Any thoughts? Thanks!

Seems like it could be related to: https://github.com/jscs-dev/node-jscs/issues/1298
Note - even when i install from `jscs-dev/node-jscs#master` this failure happens :(

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/24800842-issue-with-requirepaddingnewlinesafterblocks-inside-of-iifes?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by markelog**
Fixes #1555

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
Is this ready for review?

**Comment by markelog** [comment]
Yeah, it seems travis failure is another coveralls trick

File: test/specs/rules/require-padding-newlines-after-blocks.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var assert = require('assert');
 var Checker = require('../../../lib/checker');
 var reportAndFix = require('../../lib/assertHelpers').reportAndFix;
 
-describe('rules/require-padding-newlines-after-blocks', function() {
+describe.only('rules/require-padding-newlines-after-blocks', function() {
```
Comment (mrjoelkemp) [inline test/specs/rules/require-padding-newlines-after-blocks.js]: Remove

Comment (markelog) [inline test/specs/rules/require-padding-newlines-after-blocks.js]: Done

**Comment by mrjoelkemp** [comment]
LGTM

**Pull request closed without merging**